test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -485,15 +494,15 @@ func (co churnOp) patchParams(w *workload) (realOp, error) {
 	return &co, nil
 }
 
-// barrierOp defines an op that can be used to wait until all scheduled pods of
-// one or many namespaces have been bound to nodes. This is useful when pods
+// barrierOp defines an op that can be used to wait until all scheduled pods created by
+// one or many createPodsOp have been bound to nodes. This is useful when pods
 // were scheduled with SkipWaitToCompletion set to true.
 type barrierOp struct {
 	// Must be "barrier".
 	Opcode string
-	// Namespaces to block on. Empty array or not specifying this field signifies
-	// that the barrier should block on all namespaces.
-	Namespaces []string
+	// IDs of createPodsOp to block on. Empty array or not specifying this field signifies
```

Comment:
Do we really need this change?

It should be fine to just block based on namespaces

Reply:
you are right, I reverted this change.